controller/src/main/java/io/pravega/controller/server/rest/generated/api/RFC3339DateFormat.java

```diff
@@ -1,17 +1,15 @@
 package io.pravega.controller.server.rest.generated.api;
 
-import com.fasterxml.jackson.databind.util.ISO8601DateFormat;
-import com.fasterxml.jackson.databind.util.ISO8601Utils;
-
 import java.text.FieldPosition;
 import java.util.Date;
 
-public class RFC3339DateFormat extends ISO8601DateFormat {
+@SuppressWarnings("deprecation") //See https://github.com/FasterXML/jackson-databind/issues/1786
+public class RFC3339DateFormat extends com.fasterxml.jackson.databind.util.ISO8601DateFormat {
 
     // Same as ISO8601DateFormat but serializing milliseconds.
     @Override
     public StringBuffer format(Date date, StringBuffer toAppendTo, FieldPosition fieldPosition) {
-        String value = ISO8601Utils.format(date, true);
+        String value = com.fasterxml.jackson.databind.util.ISO8601Utils.format(date, true);
```

Comment:
why are we updating generated file?

Reply:
Otherwise the build fails with a compilation error.
There is no functional change here, this just suppresses the warnings from importing a deprecated class. Which would normally be a warning, but our build is set to treat warnings as errors and fail the build. Originally I just changed that behaviour but based on feedback, it seemed better to remove that safety net for the sake of preserving one generated file in it's unmodified state.

Once there is an update to the code generator to use non-deprecated code, we should be able to re-generate the file and leave it unmodified.

Reply:
Given the constraints in front of us, I'd say that we modify the file and create a separate issue explaining the problem and pointing to the jackson issue. The new issue should also say that once the jackson issue is resolved, we need to upgrade and regenerate the file.

Reply:
@tkaitchuck can you please open an issue for the same.

Reply:
Created https://github.com/pravega/pravega/issues/4224